### DISCUSSION TITLE
Hosting suggestions?

### DISCUSSION THREAD

**Discussion by 99hats** [lllama/dark-star#5] [General]
Hi there,
The clean routing and minimalism of Dark Star seems perfect for developing MVPs. Any suggestions on hosting? I've used a little Heroku, a little Vercel in the past, but have been mostly JAMstacking lately.

**Comment by lllama** [comment]
I've not used them but I have my eye on https://fly.io and https://render.com

**Comment by 99hats** [comment]
I just discovered Deta, a free developers hosting platform, and created a starter template for it. Worth the look, it includes a NoSQL db and file storage.

[Deta Starter template](https://github.com/99hats/darkstar-deta-starter)
